assignments/lambda-classes.js

```diff
@@ -1 +1,138 @@
 // CODE here for your Lambda Classes
+
+
+class Person {
+    constructor (props) {
+    this.name = props.name;
+    this.age = props.age; 
+    this.location = props.location; 
+  }
+  speak() {
+      return `Hello, my name is ${this.name}, I am from ${this.location}.`
+  }
+}
+
+class Instructor extends Person {
+    constructor(attr) {
+        super(attr);
+        this.specialty = attr.specialty;
+        this.favLanguage = attr.favLanguage; 
+        this.catchPhrase = attr.catchPhrase; 
+  }
+
+  demo (subject) {
+      return `Today we are learning about ${subject}.`;
+  }
+
+  grade(student, subj) {
+      return `${student} receives a perfect score on ${subj}.`;
+  }
+}
+
+
+class Student extends Person {
+    constructor(stud) {
+        super(stud);
+        this.previousBackground = stud.previousBackground;
+        this.className = stud.className; 
+        this.favSubjects = stud.favSubjects; 
+  }
+
+  listsSubjects () {
+      return `${this.favSubjects}`;
+  }
```

Comment:
If they had multiple `favSubjects` here, how would you print those out line by line?